unstructured/embed/mixedbreadai.py

```diff
@@ -0,0 +1,211 @@
+import os
```

Comment:
pls call classes mixedbread_ai instead of mixedbreadai 
(docs as well as embedder)

Reply:
no, its voyageai, openai and vertexai. And we used mixedbreadai as filenames for other integrations as well.